index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
```

Comment:
So now we need index.html at the root?

Reply:
Yes this is a specificity of Vite. The entry point is `index.html` and it should be at the root.

Reply:
See [index.html and project root in `vite.js` docs](https://vitejs.dev/guide/#index-html-and-project-root).